### PR TITLE
New functionality to change app logos

### PR DESCRIPTION
**Pull request by matiasmoreno876**
Hi Team,
In this PR I add the functionality to change app logos.
Issue [#3491](https://github.com/wazuh/wazuh-kibana-app/issues/3491)

### Steps to reproduce

1. Go to Settings/Configuration
2. Search the section `Logo Customization`
image

3. Define the name to the app logo, logo sidebar and health check. 
4. You have to add the new logos in the `/usr/share/kibana/plugins/wazuh/public/assets` path inside the Kibana container
image
example: `icon_blue_2.svg` and `logotype-2.svg`
5. Change the name and save
image

image

image
image
image

### CODE REVIEW

File: public/plugin.ts [hunk 1]
```diff
@@ -64,6 +64,12 @@ export class WazuhPlugin implements Plugin<WazuhSetup, WazuhStart, WazuhSetupPlu
...
         //Update if user has Wazuh disabled
         this.stateUpdater.next(() => {
+          return { category: {
+              id: 'wazuh',
+              label: 'Wazuh',
+              order: 0,
+              euiIconType: core.http.basePath.prepend( `/plugins/wazuh/assets/${response.logoSidebar}`),
+            }}
           if(response.isWazuhDisabled) unmount();
           return { status: response.isWazuhDisabled }
```
Comment (gabiwassan) [inline public/plugin.ts:74]: Here you have 2 returns 🤔

**Review by gabiwassan — CHANGES_REQUESTED** [review]

File: public/plugin.ts (hunk 1 above)
Comment (matiasmoreno876) [reply public/plugin.ts]: Solved!

File: public/utils/config-equivalences.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 export const configEquivalences = {
   pattern: 'Default index pattern to use on the app.',
+  'customization.logo.app':'Define the name of the app logo saved in the path /plugins/wazuh/assets/',
+  'customization.logo.sidebar':'Define the name of the sidebar logo saved in the path /plugins/wazuh/assets/',
+  'customization.logo.healthcheck':'Define the name of the health-check logo saved in the path /plugins/wazuh/assets/',
+  'customization.logo.reports':'Define the name of the reports logo (.png) saved in the path /plugins/wazuh/assets/',
```
Comment (frankeros) [inline public/utils/config-equivalences.js:6]: couldn't it be any type of image (png,svg,jpg,etc)?

**Review by frankeros — APPROVED** [review]
LGTM!

**Review by gabiwassan — COMMENTED** [review]
LGTM! 👍🏼

File: public/utils/config-equivalences.js (hunk 2 above)
Comment (matiasmoreno876) [reply public/utils/config-equivalences.js:6]: Currently, if the `.png` format is not used, it breaks when generating the report.

**Review by gabiwassan — APPROVED** [review]

**Comment by juliannediaz12** [comment]
Hi! A few weeks ago I updated my test environment and got 4.2.2 and it worked. And just today upgraded our prod manager from 4.2.1 to 4.2.4(recent) to change the logo and the option to change the logo was not there anymore. So I tried to edit the config file instead but my added variables did not take effect. I already restarted both kibana and wazuh. Was the logo customization removed?

**Pull request merged**